Memorize/MemoryGame.swift

```diff
@@ -8,79 +8,81 @@
 import Foundation
 
 struct MemoryGame<CardContent> where CardContent: Equatable {
-
     // MARK: - Properties
+    
     struct Card: Identifiable {
-
         var isFaceUp: Bool = false
         var isMatched: Bool = false
         var content: CardContent
         var isPreviouslySeen: Bool = false
         var id: Int
-
     }
-
     private(set) var cards: [Card]
-//    private var indexOfTheOneAndOnlyFaceUpCard: Int?
     private var indexOfTheOneAndOnlyFaceUpCard: Card?
     private(set) var score: Int = 0
-
+    
     // MARK: - Initilization
+    
     init(numberOfPairsOfCards: Int, createCardContent: (Int) -> CardContent) {
         cards = [Card]()
         for pairIndex in 0..<numberOfPairsOfCards {
             let content: CardContent = createCardContent(pairIndex)
             cards.append(Card(content: content, id: pairIndex*2))
-            cards.append(Card(content: content, id: pairIndex * 2 + 1))
+            cards.append(Card(content: content, id: pairIndex*2 + 1))
         }
         cards.shuffle()
     }
-
+    
     // MARK: - Functions
+    
     mutating func choose(_ card: Card) {
+        """
+        This function is used to update the score and card details to statisy the following game conditions:
+        1. At a time only two cards should be faceUp
+        2. If the content of the card is matching then update the score by 2
+        3. else facedown all the cards and if the if the any one of the card is already seen then reduce score by 1
+        """
+        
```

Comment:
That's not a docstring in Swift.  For example, if you Option-Click on `choose` in Xcode, what do you see?

Please correct here and throughout.  (Search your project for `"""` to see all the other places.)

Here is an example of a docstring in Swift:
![image](https://user-images.githubusercontent.com/4765449/188007263-a6a42e9a-e87e-475c-822d-8b2b8996d6e0.png)
If, then, you were to Option-Click on `MeasurementSytem` in Xcode, you would see this:
![image](https://user-images.githubusercontent.com/4765449/188007371-2a668de6-ef73-4ee8-aa49-a76394dbbf28.png)